gax-grpc/src/main/java/com/google/api/gax/grpc/OperationCallable.java

```diff
@@ -48,77 +48,76 @@
 @BetaApi
 public final class OperationCallable<RequestT, ResponseT extends Message> {
   private final UnaryCallable<RequestT, Operation> initialCallable;
-  private final Channel channel;
-  private final ScheduledExecutorService executor;
+  private final ClientInitContext clientContext;
```

Comment:
This is the spot that makes the name `ClientInitContext` weird - it is referenced after initialization.